src/core/model/uischema.ts

```diff
@@ -64,7 +66,7 @@ const traverse = (
   parent?: LinkedUISchemaElement
 ): void => {
   pre(uiSchema, parent);
-  if (isLayout(uiSchema)) {
+  if (uiSchema && isLayout(uiSchema)) {
```

Comment:
shouldn't isLayout handle the null/undefined case?

Reply:
the check is correct, the JSON Forms' `isLayout` doesn't check for undefined.

Reply:
Fix jsonforms then ;-)